.travis.yml

```diff
@@ -1,14 +1,22 @@
 language: java
 sudo: false
+
 cache:
   directories:
   - .autoconf
   - $HOME/.m2
+  - $HOME/.zinc
+
+script: mvn scoverage:report
```

Comment:
Not sure if this intended or not and it doesn't seem like it makes a difference, but travis now seems to run the tests as a side-effect of the scoverage run rather than running the tests explicitly?

before travis ran:
```
mvn install -DskipTests=true -Dmaven.javadoc.skip=true -B -V
mvn test -B
```

now it runs

```
mvn install -DskipTests=true -Dmaven.javadoc.skip=true -B -V
mvn scoverage:report
```